packages/web/test/mocks/myalgowallet-mock.ts

```diff
@@ -0,0 +1,91 @@
+import type {
+    Options,
+    Accounts,
+    Address,
+    AlgorandTxn,
+    Base64,
+    ConnectionSettings,
+    EncodedTransaction,
+    SignedTx,
+    SignTransactionOptions,
+} from "@randlabs/myalgo-connect";
+
+export default class MyAlgoConnectMock {
+
+    /**
+     * @param {Options} options Override default popup options.
+     */
+    constructor(options?: Options) {
```

Comment:
Why to even create it if you don't wanna use it?

Reply:
we need to create a mock to be used as connector to connect to wallet, thus we need to create a dummy class for it